README.md

```diff
@@ -44,6 +44,11 @@ After installing angular-cli we need to install our required node modules, so we
 ```sh
 $ npm install
 ```
+After installing node modules we need to install `webpack`, so we will do that by using the following command:
+```sh
+$ npm install webpack
```

Comment:
I am not sure if we require `webpack` as a project dependency. If so, why isn't is already there in the `package.json` file?

cc @hemantjadon @Achint08.

Reply:
I gave you the logs. I will check for this in `package.json`